tests/core/test_operator/test_document_diff.py

```diff
@@ -0,0 +1,20 @@
+from copy import deepcopy
+import json
+import random
+
+from workflows_core.operator.abstract_operator import AbstractOperator
+
+
+class TestDocumentDiff:
+    def test_diff(self):
+        old_documents = [
+            {"example_vector_": [random.random() for _ in range(5)]} for _ in range(5)
```

Comment:
This doesn't actually test what is required.

Reply:
huh? Im confused, we want the difference between both documents to remove the common vector between them